stretch-assignment/index.js

```diff
@@ -0,0 +1,56 @@
+const red = document.querySelector('.block--red');
+const blue = document.querySelector('.block--blue');
+const green = document.querySelector('.block--green');
+const pink = document.querySelector('.block--pink');
+const gray = document.querySelector('.block--gray');
+
+
```

Comment:
So I spent a few hours trying to find a different way to do this, but after everything I couldn't really figure out a different way. But, we can definitely make those code a little DRYer.

How to turn 56 lines of code into 9 lines:

```suggestion
let individualBlocks = document.querySelectorAll(".block");

let blockOrder = 0;
individualBlocks.forEach((block, i) => {
  block.addEventListener("click", e => {
    e.target.style.order = blockOrder;
    --blockOrder;
  });
});
```